packages/rxsocketio/README.md

```diff
@@ -28,3 +28,28 @@ yarn add @bottlenose/rxsocketio
 ## Documentation & Guides
 - [Documentation & Guides](https://buccaneerai.gitbook.io/bottlenose/sources-and-sinks/rxsocketio)
 
+## Basic Usage
+```js
+import {from} from 'rxjs';
+import {conduit} from '@bottlenose/rxsocketio';
+
+const messageIn$ = from([
+  {topic: 'message', body: 'yarrr'},
+  {topic: 'message', body: 'arrr matey'},
+  {topic: 'message', body: 'Vitamin C? Never heard of it.'},
+]);
+
+const socketConfig = {
+  url: 'http://localhost:9080/ws', // socket.io server
+  topics: ['message'], // topics to subscribe to. (Defaults to ['message']).
+}; 
+// the conduit operator sends messages from messageIn$ and emits messages 
+// from the server
+const messageBack$ = messageIn$.pipe(conduit({...socketConfig}));
+messageBack$.subscribe(console.log);
+// {topic: 'message', body: 'Welcome Matey.'}
+// {topic: 'message', body: 'Yo ho. Yo ho. I am a message from the server.'}
+```
+
+The library also supports some advanced features commonly needed in real applications (like sending binary, verifying receipt of messages before sending the next, customizing the socket.io client, serializers/deserializers and handling disconnections).  See the documentation for more information.
+
```

Comment:
Improved docs since this package should be ready to release soon.  At that point, I might write a short blog post about it so that @bottlenose can start to accumulate Github stars...  I think we're actually starting to have a pretty good collection of useful operators

Reply:
I agree!